modules/nwt-gateway-mode.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+
+:_mod-docs-content-type: PROCEDURE
+[id="nwt-gateway-mode_{context}"]
+= Setting local and shared gateway modes
+
+As a cluster administrator you can configure the gateway mode using the `gatewayConfig` spec in the Cluster Network Operator. The following procedure can be used to set the `routingViaHost` field to `true` for local mode or  `false` for shared mode.
+
+You might follow the optional step 4 to enable IP forwarding alongside local gateway mode if you require node level traffic control. For example, possible use cases for combining local gateway mode with IP forwarding include:
```

Comment:
Enabling IP forwarding toes not allow "node level traffic control", so that sentence should be removed. It allows the host network of the node to act as router for traffic not related to OVN-Kubernetes.

Reply:
updated